pkgs/README.md

```diff
@@ -384,7 +384,13 @@ All versions of a package _must_ be included in `all-packages.nix` to make sure
 * `meta.license` must be set and match the upstream license.
   * If there is no upstream license, `meta.license` should default to `lib.licenses.unfree`.
   * If in doubt, try to contact the upstream developers for clarification.
-* `meta.mainProgram` must be set when appropriate.
+* `meta.mainProgram` must be set when there is any main executable for the package in `$bin/bin` or `$out/bin`. Set it to a string naming the executable in that directory.
```

Comment:
I find it a bit confusing or even unhelpful to define a the meaning of a composite word using just the words its made of or direct synonyms thereof.

A user coming to this doc is likely to look for guidance what exactly "main program" means and what it doesn't.

What do you think of something like this?

```Suggestion
* `meta.mainProgram` must be set to the executable name of the program which facilitates the primary function or purpose of the package. Set it to the executable name found in `$bin/bin/` or `$out/bin/`.  
  If there is no one program which represents the package's primary function (i.e. there are multiple or none), do not set this value. 
```